addons/website_event_exhibitor/views/event_sponsor_views.xml

```diff
@@ -173,6 +173,11 @@
                 <templates>
                     <t t-name="kanban-box">
                         <div class="oe_kanban_global_click overflow-hidden">
+                            <div class="o_kanban_card_header">
```

Comment:
Hello @thsh-odoo Do we really need this?

Reply:
no, it's not necessary 
removed it :smiley: